### PR TITLE
initial implementation of group controller

### PR DESCRIPTION
**Pull request by Feliscatus8**

### CODE REVIEW

File: src/main/java/com/kodilla/ecommercee/GroupController.java [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/group")
+@CrossOrigin("*")
 public class GroupController {
```
Comment (gawrysiakg) [inline src/main/java/com/kodilla/ecommercee/GroupController.java]: Maybe you should put this GroupController in the appropriate controller folder and GroupDto in domain/dto. What do you think?

File: src/main/java/com/kodilla/ecommercee/GroupDto.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.kodilla.ecommercee;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
```
Comment (gawrysiakg) [inline src/main/java/com/kodilla/ecommercee/GroupDto.java]: These two imports seem to be unused in Dto, but it's not a problem.

**Review by gawrysiakg — APPROVED** [review]

**Review by andwis93 — APPROVED** [review]

File: src/main/java/com/kodilla/ecommercee/GroupController.java (hunk 1 above)
Comment (Feliscatus8) [reply src/main/java/com/kodilla/ecommercee/GroupController.java]: yes it is a good idea to add some structore

File: src/main/java/com/kodilla/ecommercee/GroupDto.java (hunk 2 above)
Comment (Feliscatus8) [reply src/main/java/com/kodilla/ecommercee/GroupDto.java]: thanks i didn't notice it

**Review by viepovsky — APPROVED** [review]

**Pull request merged**